flake.nix

```diff
@@ -1,40 +1,36 @@
 {
-  description = "SAC development environment";
+  description = "SaC Compiler and Stdlib";
 
-  inputs.nixpkgs.url = github:NixOS/nixpkgs/nixpkgs-unstable;
-  inputs.flake-utils.url = github:numtide/flake-utils;
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs";
+    flake-utils.url = "github:numtide/flake-utils";
```

Comment:
As the `forEeachSystem` was removed, you can scrap this dependency. I guess mac support is utopic to assume from this nix setup (I at least didn't attempt it and had no way to test it anyway)

Reply:
We do have a MacOS package... but how to install that with NixOS I do not know. I suspect it won't work, as we package it into a PKG format, which is unique to MacOS. I do not know if NixOS supports exporting/extracting this format.

I'll remove the dependency for the moment.